packages/react/src/utils.js

```diff
@@ -1,4 +1,2 @@
 // @flow
 export let isBrowser = typeof document !== 'undefined'
-
-export const hasOwnProperty = {}.hasOwnProperty
```

Comment:
Instead of removing this let's rename this to `hasOwnProp` or `hasOwn` (since there is a static `Object.hasOwn` the latter might be the appropriate name)

Reply:
Why we should keep hasOwn in utils? I have replaced all `hasOwnProperty` mentions with `hasOwn`

Reply:
```
'use strict'
Object.freeze(Object.prototype)
var exports = {}
// exports.hasOwnProperty = Object.hasOwnProperty
// exports.hasOwnProperty = () => 'injected!'
exports.hasOwn = () => 'injected!'
Object.hasOwn(Object.prototype, 'hasOwnProperty')
// true
```

Reply:
> Why we should keep hasOwn in utils?

To avoid problems with `{ hasOwnProperty: 'whatever' }`. 

> I have replaced all hasOwnProperty mentions with hasOwn

It's a fairly new addition to the language and it's not supported by all of our targets so we can't use it.

Reply:
Can we replace it with `Object.hasOwnProperty.call`?

`export const hasOwn = Object.hasOwnProperty.call`

Reply:
Or `export const hasOwn = Object.prototype.hasOwnProperty?.call || () => 'not sure about that - target do not support hasOwnProperty method'`